src/modules/yutobot/index.js

```diff
@@ -75,6 +83,29 @@ const start = () => {
 						}
 						message.channel.send(owoifierEnabled ? "1" : "0");
 					}
+				
+				
+				case "Ⓐ":
+					//vars for time/date
+					ctx.textAlign = "center";
+					var dateObj = new Date()
```

Comment:
Add a semicolon at the end of every statement to prevent weird javascript shenanigans (probably irrelevant here but it's a good practice :))